com.unity.render-pipelines.universal/Editor/ShaderGraph/Includes/DepthNormalsOnlyPass.hlsl

```diff
@@ -24,14 +24,29 @@ half4 frag(PackedVaryings packedInput) : SV_TARGET
     #endif
 
     #if defined(_GBUFFER_NORMALS_OCT)
-    float3 normalWS = normalize(unpacked.normalWS);
-    float2 octNormalWS = PackNormalOctQuadEncode(normalWS);           // values between [-1, +1], must use fp32 on some platforms
-    float2 remappedOctNormalWS = saturate(octNormalWS * 0.5 + 0.5);   // values between [ 0,  1]
-    half3 packedNormalWS = PackFloat2To888(remappedOctNormalWS);      // values between [ 0,  1]
-    return half4(packedNormalWS, 0.0);
+        float3 normalWS = normalize(unpacked.normalWS);
+        float2 octNormalWS = PackNormalOctQuadEncode(normalWS);           // values between [-1, +1], must use fp32 on Nintendo Switch.
```

Comment:
Is this a bad merge? it seems like you are reverting changes to the comment

Reply:
Yes and no. Only the comment seems to be incorrect here, the code is the same (apart from me indenting things) from the original.
Will fix the comment in the next commit.